vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -48,9 +48,24 @@ void draw_pkg_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     if (draw_file_dialog) {
         result = host::dialog::filesystem::open_file(pkg_path, { { "PlayStation Store Downloaded Package", { "pkg" } } });
         draw_file_dialog = false;
-        if (result == host::dialog::filesystem::Result::SUCCESS)
-            ImGui::OpenPopup("install");
-        else if (result == host::dialog::filesystem::Result::CANCEL) {
+        if (result == host::dialog::filesystem::Result::SUCCESS) {
+            fs::ifstream infile(pkg_path.string(), std::ios::binary);
```

Comment:
```suggestion
            fs::ifstream infile(pkg_path, std::ios::binary);
```
Here too. I forgot.

Reply:
🤯

Reply:
Sorry, I was wrong. For filesystem.path it's better to use fs::path(pkg_path.native())
```
fs::ifstream infile(fs::path(pkg_path.native()), std::ios::binary);
//or
fs::ifstream infile(pkg_path.native(), std::ios::binary);
```
converting path to string can cause crash on some old systems (early versions of win10 and before) for unicode paths